ios/FluentUI/People Picker/ContactView.swift

```diff
@@ -106,12 +115,22 @@ open class ContactView: UIView {
             constraints.append(contentsOf: identifierLayoutConstraints())
         }
         titleLabel.translatesAutoresizingMaskIntoConstraints = false
-        labelContainer.addSubview(titleLabel)
+
+        pressedStateOverlay.translatesAutoresizingMaskIntoConstraints = false
+        avatarView.addSubview(pressedStateOverlay)
+        constraints.append(contentsOf: [
+            pressedStateOverlay.leadingAnchor.constraint(equalTo: avatarView.leadingAnchor),
+            pressedStateOverlay.trailingAnchor.constraint(equalTo: avatarView.trailingAnchor),
+            pressedStateOverlay.topAnchor.constraint(equalTo: avatarView.topAnchor),
+            pressedStateOverlay.bottomAnchor.constraint(equalTo: avatarView.bottomAnchor)
+        ])
 
         labelContainer.translatesAutoresizingMaskIntoConstraints = false
+        labelContainer.isUserInteractionEnabled = false
```

Comment:
Are these isUserInteractionEnabled calls useful? Curious to know why.

Reply:
The `isUserInteractionEnabled` for both the `labelContainer` and `avatarView` are needed for the `ContactView` to receive the touch events. When `isUserInteractionEnabled` is set to true (i.e. by default), the `ContactView` is not able to receive any touch events and the handlers in turn will not be called.